### DISCUSSION TITLE
2 LXQt issues Power save to monitors and QT6

### DISCUSSION THREAD

**Discussion by LargePrime** [lxqt/lxqt#2221] [Q&A]
OS:Artix openrc LXQt
well here goes. Artix . I have several ongoing issues.
1) https://bayimg.com/MABgGAaHA qt6 looks like this. if I hit apply, without changing anything on the config tool it looks like this https://bayimg.com/mabghaAha If I close the config tool it reverts
2) power save keeps happening (monitor shut off) even tho disabled in LXQt settings I think this is the issue https://www.x.org/releases/X11R7.7/doc/xextproto/dpms.html

**Comment by tsujan** [comment]
"Qt6 Configuration Tool" doesn't belong to Qt or LXQt. It's a separate platform plugin for Qt6.

LXQt isn't ported to Qt6 yet. When it becomes ported, you'll have to either disable `qt6ct`or tolerate its conflict with LXQt's Qt6 plugin.

EDIT: You might want to ask qt6ct's developer(s) about your problem.

**Comment by tsujan** [comment]
As for Qt6 port, see https://github.com/lxqt/lxqt/discussions/2139

**Comment by LargePrime** [comment]
I see you have done alot with LXQt and QT6.

It was working fine for months. now it has this issue. I post here too https://github.com/trialuser02/qt6ct/issues/14

Any thoughts on the screen shutting off issue?

**Reply by tsujan** [reply]
> Any thoughts on the screen shutting off issue?

See https://wiki.archlinux.org/title/Display_Power_Management_Signaling#Disabling_DPMS

**Reply by tsujan** [reply]
> It was working fine for months. now it has this issue

After recent Qt6 upgrades, it may need recompilation or code change. But that's just a wild guess; its developer could give you the best answer.

EDIT: With the way you reported the problem at https://github.com/trialuser02/qt6ct/issues/14, I don't think any dev would give it priority  ;)

**Reply by LargePrime** [reply]
It seems that dev is rather non responsive in general, no?

**Reply by LargePrime** [reply]
Should LXQt take over the power management, and disable DPMS, as it's power management suggests it does?

**Reply by tsujan** [reply]
→ /usr/share/lxqt/power.conf
